docker/config.sh

```diff
@@ -6,6 +6,7 @@ if [ -d "$DIR" ]; then
   cp "$DIR"/* .
   # needed because it is hidden
   cp "$DIR"/.env .
+  mv ./config.features.php features/
```

Comment:
Is this needed? Does the `cp` not take care of all files and subfolders?

Reply:
My mistake, I didn't notice the features folder in `appconfig`